openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -536,7 +536,7 @@ def _process_texture(self, filepath, resource,
                                             color_space,
                                             render_colorspace])
                 else:
-
+                    config_path = ""
```

Comment:
This is fine to be `None` I'd say since we're checking with if statement before we use it anyway?